msg/Status.msg

```diff
@@ -11,6 +11,13 @@ bool error_status
 uint16 error_code
 # Does the laser report that it is locked out.
 bool lockout_status
+# State of the different warning and safety zones
+bool ossd_1
+bool ossd_2
+bool warning_1
+bool warning_2
+bool ossd_3
+bool ossd_4
```

Comment:
Is there a particular reason to have warning_1/2 in between ossd_2/3?

Reply:
I mimicked the order in which the information get transmitted by the scanner to the PC, see attached image. But it can also be reorded to have warning_1/2 first and then ossd, if you prefer.
![Screenshot from 2023-05-26 09-19-06](https://github.com/ros-drivers/urg_node_msgs/assets/15085346/4b622486-f881-4aa0-bd45-fbe8d1810192)

Reply:
Same order as the packet is good enough reason for me